src/renderer/components/AddonSection/index.tsx

```diff
@@ -254,11 +176,10 @@ export const AddonSection = (): JSX.Element => {
   }, [dispatch, publisherData, selectedAddon]);
 
   useEffect(() => {
-    findInstalledTrack();
     if (!isInstalling) {
-      InstallManager.determineAddonInstallState(selectedAddon).then(setCurrentInstallStatus);
+      void InstallManager.getAddonInstallState(selectedAddon);
     }
-  }, [findInstalledTrack, isInstalling, selectedAddon, setCurrentInstallStatus]);
+  }, [isInstalling, selectedAddon]);
```

Comment:
Is this function still supposed to refresh the state when changing the addon, like it did previously? It doesn't anymore if a state exists (which basically has to based on App\index.tsx line 36-39)

Reply:
Fixed.